pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/stage0-posix-x86.nix

```diff
@@ -0,0 +1,345 @@
+# This is a translation of stage0-posix/stage0-posix/x86/mescc-tools-mini-kaem.kaem to nix
```

Comment:
It would be cute, but not required, if the final PR to be merged had a commit which vendors the scheme files and then a second commit which "edits" them to turn them into Nix code.  Just for posterity.